api/tests/views_tests.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+
+import pytest
+
+@pytest.mark.parametrize("format", ("yaml", "json"))
+def test_swagger_format(client, format):
+    path = f"/api/v1/swagger.{format}"
+    response = client.get(path)
+    assert response.status_code == 200
+    expected_content_type = {
+        "yaml": "application/yaml; charset=utf-8",
+        "json": "application/json; charset=utf-8",
+    }[format]
+    assert response["Content-Type"].startswith(f"application/{format}")
```

Comment:
`expected_content_type` is unused? Could this maybe be ...

```suggestion
    expected_media_type = {
        "yaml": "application/yaml",
        "json": "application/json",
    }[format]
    assert response["Content-Type"].startswith(expected_media_type)
```

or we just drop it altogether?

```suggestion
    assert response["Content-Type"].startswith(f"application/{format}")
```

Reply:
yes, `expected_content_type` is garbage from previous iterations of getting the test to work. I think dropping it is the right thing, keeps the test short.